lib/gherkin_lint/linter/required_tags_starts_with.rb

```diff
@@ -0,0 +1,20 @@
+require 'gherkin_lint/linter/tag_constraint'
+require 'gherkin_lint/linter'
+
+module GherkinLint
+  # service class to lint for tags used multiple times
+  class RequiredTagsStartsWith < Linter
+    include TagConstraint
+
+    def match_pattern?(target)
+      match = false
+
+      target.each do |t|
+        t.delete! '@'
+        match = t.start_with?(*@pattern)
+        break if match
```

Comment:
This is implemented in the mixin.  I have some ideas about how to change 'add_error' but I'd rather not overwrite lint in the child simply to change the error message.  The idea would be add DebugLevel as a member in each linter config, set the defaults as you have and then have a more generic error mixin that would call each linters 'print_error' method.  That gives you more control